Doc/c-api/stable.rst

```diff
@@ -147,12 +160,12 @@ This is the case with Windows and macOS releases from ``python.org`` and many
 third-party distributors.
 
 
-.. _stable-abi-list:
+.. _limited-api-list:
```

Comment:
This looks weird; there's already such an anchor in line 171.

Reply:
Ah, a _similar_ anchor in 171; not equal.

Reply:
There are 4 cross-references (one in ``stable.rst`` (L40), one in 3.11 whatsnew, and two in the 3.11 changelog) -- search for ``[`<]limited-api-list[`>]``.

A

Reply:
> Ah, a *similar* anchor in 171; not equal.

Oh, I think I understand. L171 is a custom directive also named ``limited-api-list``, to enumerate the limited API -- see [the rendered page](https://docs.python.org/dev/c-api/stable.html#contents-of-limited-api) for an example. So both are valid.

Reply:
Ok, thanks!